### PR TITLE
correct idiom 248

### PR DESCRIPTION
**Pull request by simonvonhackewitz**
the old implementation was based on the incorrect python implementation

### CODE REVIEW

**Review by michaellilltokiwa — APPROVED** [review]

File: src/idiommainex.fz.expected_err
Comment (michaellilltokiwa) [inline src/idiommainex.fz.expected_err:1]: @simonvonhackewitz what are those files? Committed by accident?

**Review by michaellilltokiwa — CHANGES_REQUESTED** [review]
see comment

File: src/idiommainex.fz.expected_err
Comment (simonvonhackewitz) [reply src/idiommainex.fz.expected_err:1]: yes 🙈
I had a script for committing idioms, but it did not work under these circumstances.

Comment (michaellilltokiwa) [reply src/idiommainex.fz.expected_err:1]: No worries. :-)

**Pull request merged**